hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/OmMetadataGenerator.java

```diff
@@ -309,7 +322,7 @@ public Supplier<String> realTimeStatusSupplier() {
     };
   }
 
-  @SuppressWarnings("checkstyle:EmptyBlock")
+  @SuppressWarnings({"checkstyle:EmptyBlock", "checkstyle:MethodLength"})
```

Comment:
Instead, we can extract pieces from this giant method to smaller functions....

Reply:
Code has been simplified.